lib/tasks/update_vacancy_contract_type.rake

```diff
@@ -0,0 +1,16 @@
+namespace :vacancies do
+  desc "Update vacancies with contract_type == parental_leave_cover to contract_type == fixed_term and set is_parental_leave_cover to true"
+  task update_parental_leave_cover: :environment do
+    vacancies_to_update = Vacancy.where(contract_type: Vacancy.contract_types[:parental_leave_cover])
+
+    vacancies_to_update.find_each do |vacancy|
+      vacancy.update(
+        contract_type: Vacancy.contract_types[:fixed_term],
+        is_parental_leave_cover: true,
+        fixed_term_contract_duration: vacancy.parental_leave_cover_contract_duration,
```

Comment:
Question: There will be a follow-up PR removing `parental_leave_cover_contract_duration` from DB?
It seems this field won't be used anymore after the rake task execution.

Reply:
Yep! I would also like to be sure that everything is working as expected before we delete data that I think would be hard to recover